src/algebra/pointwise.lean

```diff
@@ -1,153 +1,136 @@
 /-
 Copyright (c) 2019 Johan Commelin. All rights reserved.
 Released under Apache 2.0 license as described in the file LICENSE.
-Authors: Johan Commelin
+Authors: Johan Commelin, Floris van Doorn
 -/
 import algebra.module
 
 /-!
 
 # Pointwise addition, multiplication, and scalar multiplication of sets.
 
-This file defines `pointwise_mul`: for a type `α` with multiplication,
-multiplication is defined on `set α` by taking `s * t` to be the set
-of all `x * y` where `x ∈ s` and `y ∈ t`.
+This file defines pointwise algebraic operations on sets.
+* For a type `α` with multiplication, multiplication is defined on `set α` by taking
+  `s * t` to be the set of all `x * y` where `x ∈ s` and `y ∈ t`.
```

Comment:
Please mention the additive version here. Otherwise it's not clear why we need `set_semiring`.